### PR TITLE
[italian] Translated welcome/install-d-locally.md

### PR DESCRIPTION
**Pull request by lodo1995**
@trikko Could you please review this?
Of course there's no hurry, do it when you have the time and tools to do so. It does not block further work from happening.
Thank you very much for your effort.

### CODE REVIEW

File: public/content/it/welcome/install-d-locally.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* Tramite il [programma di installazione](http://downloads.dlang.org/releases/2.x/2.071.0/dmd-2.071.0.exe)
+* oppure come [archivio compresso](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.windows.7z)
+* o tramite [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* Come [pacchetto `.dmg`](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.dmg)
+* oppure come [archivio compresso](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.osx.tar.xz)
+* o tramite [Homebrew](http://brew.sh): `brew install dmd`
```
Comment (wilzbach) [inline public/content/it/welcome/install-d-locally.md]: I just realized that we are hard-coding the latest release here, that will be fixed very soon (#449).
So how about applying `s/2.071.0/{{latest-release}}/g` with your favorite regex tool (e.g. `sed s/2.071.0/{{latest-release}}/g  -i <file> `)?

**Comment by wilzbach** [comment]
@lodo1995 this is the only pending PR for Italian, so at least new PRs should be made against the italian repository:

https://github.com/dlang-tour/italian

I can make a "submodule" PR once this PR has been merged, but of course you can do this yourself too ;-)

**Comment by lodo1995** [comment]
@wilzbach thank you for all your work. I'm closing this. I will reopen it in the new repository (probably next week).
I don't know how you want to proceed with the other parts. I can easily re-upload them to the italian repository (again, next week), so I'd suggest you open a PR that removes them from the base repo and sets up the connection to the italian repo (as you did for the english one).

**Comment by wilzbach** [comment]
> so I'd suggest you open a PR that removes them from the base repo and sets up the connection to the italian repo (as you did for the english one).

Alrighty, done :)

https://github.com/dlang-tour/italian/pull/1
https://github.com/stonemaster/dlang-tour/pull/462

(moving the files over was quick, so I already did that)

**Comment by lodo1995** [comment]
> Alrighty, done :)

Thank you very much.

**Pull request closed without merging**